infrastructure/ansible/roles/traffic-monitor/tasks/traffic_monitor.yml

```diff
@@ -45,6 +45,28 @@
   notify: Restart Traffic Monitor
   become: yes
 
+- name: Copy HTTPS/SSL key to specified directory
+  copy:
+    content: "{{ tm_traffic_ops_https_key }}"
+    dest: "{{ tm_traffic_ops_key_file_location }}"
+    mode: 0400
+    owner: traffic_monitor
+    group: traffic_monitor
+  notify: Restart Traffic Monitor
+  become: yes
+  when: tm_traffic_ops_use_https_listener == True
+
+- name: Copy HTTPS/SSL certificate to specified directory
+  copy:
+    content: "{{ tm_traffic_ops_https_cert }}"
+    dest: "{{ tm_traffic_ops_cert_file_location }}"
+    mode: 0400
+    owner: traffic_monitor
+    group: traffic_monitor
+  notify: Restart Traffic Monitor
+  become: yes
+  when: tm_traffic_ops_use_https_listener == True
+
```

Comment:
I don't think it's in the best interests of OSS contributors to dictate how key material is distributed.  Instead, it would be appropriate to assert it's existence however.